models/docs/sources/history_ledgers.md

```diff
@@ -108,3 +108,15 @@ The number of failed transactions submitted to the network in this ledger. The t
 {% docs tx_set_operation_count %}
 The total number of operations in the transaction set for this ledger, including failed transactions.
 {% enddocs %}
+
+{% docs soroban_fee_write_1kb %}
+Add description
```

Comment:
The write fee is dynamic fee that grows linearly with the size of the ledger. The write fee for a given ledger is recorded in the ledger header and has a direct relationship to the bucketlist size of the ledger db.